experimental/cache_size_test.cc

```diff
@@ -0,0 +1,49 @@
+
+#include "cache_size.h"
+
+#include <iostream>
+// Measure how often measuring time is able to accurately determine cache hits
+// and cache misses based on the buffer size under analysis and the cache size
+// it checks whether buffer sizes that fit in cache have less reading time or
+// not
+int main(int argc, char* argv[]) {
+  static constexpr long kCacheSize = 8 * 1024 * 1024;
```

Comment:
So different CPUs have different cache sizes, right? What makes this test work even with a cache that is smaller or bigger than MB?

My feeling is that `candidates` should probably only have big multipliers so as to ensure that candidate_small is always within cache, and candidate_large is always bigger.

Reply:
changed the weight to 20 instead of 2